src/ore/src/tracing.rs

```diff
@@ -320,21 +321,20 @@ where
         //
         // Note: folks should feel free to add more crates here if we find more
         // with long lived Spans.
-        let default_targets = [("h2", LevelFilter::OFF), ("hyper", LevelFilter::OFF)];
+        let default_directives: Vec<Directive> = vec![
+            Directive::from_str("h2=off").expect("valid directive"),
+            Directive::from_str("hyper=off").expect("valid directive"),
+        ];
 
         let (filter, filter_handle) = reload::Layer::new(if otel_config.start_enabled {
-            let new_targets = Targets::default()
-                .with_targets(default_targets)
-                .with_targets(&otel_config.filter);
-
-            if let Some(default_level) = otel_config.filter.default_level() {
-                new_targets.with_default(default_level)
-            } else {
-                new_targets
+            let mut filter = EnvFilter::from_str(&format!("{}", otel_config.filter)).expect("WIP");
+            for directive in &default_directives {
+                filter = filter.add_directive(directive.clone());
```

Comment:
what happens if a later directive overrides one of the `h2/hyper` ones? It doesnt really matter, just curious

Reply:
Good point... from the docs:

```
If a filter directive is inserted that matches exactly the same spans
and events as a previous filter, but sets a different level for those
spans and events, the previous directive is overwritten.
```

so I think we want to insert the defaults first, and then let the user-specified filters have the chance to override those. Unfortunately... there's really no good way to merge `EnvFilter`s together, so it's not obvious to me how to do this... hm

Reply:
okay I just tried it out and the user-provided values definitely take precedence over the defaults. just gonna pretend I never saw that doc comment and not think too hard 🙃